project2.css

```diff
@@ -0,0 +1,396 @@
+header {
+  position: absolute;
+  width: 1340px;
+  height: 1246px;
+  left: 200px;
+  top: 71px;
+}
+
+.grid-container {
+  display: grid;
+  justify-content: space-evenly;
+  grid-template-columns: 50px 50px 50px; /* Make the grid smaller than the container */
+  gap: 10px;
+  background-color: #fff;
+  padding: 10px;
+}
```

Comment:
to make your project seem closer to [the Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) I recommend you to use [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), as it could help you a lot with layout :smile: